src/instructlab/model/evaluate.py

```diff
@@ -191,6 +198,37 @@ def qa_pairs_to_qna_to_avg_scores(qa_pairs: list[dict]) -> dict:
     return qna_to_avg_scores
 
 
+def launch_server(
+    ctx,
+    model,
+    model_name,
+    max_workers,
+) -> tuple:
+    # pylint: disable=import-outside-toplevel
+    # First Party
+    from instructlab.model.backends import backends
+
+    if ctx.obj.config.evaluate_serve.backend == backends.VLLM:
+        ctx.obj.config.evaluate_serve.vllm.vllm_args = (
+            f"--served-model-name {model_name}"
+        )
+    if ctx.obj.config.evaluate_serve.backend == backends.LLAMA_CPP:
+        ctx.obj.config.evaluate.mt_bench.max_workers = min(max_workers, 16)
```

Comment:
So if the user passes a `max_workers` larger than 16, 16 will be used - why is that?

Reply:
There is probably still some work to do here, but llama-cpp doesn't handle the parallelism the same and vllm and starts to drop requests under too much load.  Will add a comment.

Reply:
Added comment, thanks!

Reply:
Thank you!